src/extension.node.ts

```diff
@@ -169,7 +170,9 @@ async function activateUnsafe(
 
         const [serviceManager, serviceContainer] = initializeGlobals(context);
         activatedServiceContainer = serviceContainer;
-        initializeTelemetryGlobals(serviceContainer);
+        initializeTelemetryGlobals((interpreter) =>
```

Comment:
Just pass a callback instead of passing the entire interface, as we dont want the telemetry layer to have any dependencies on interpreter code.